Assets/HoloToolkit/Utilities/Scripts/Utils.cs

```diff
@@ -10,14 +10,10 @@ namespace HoloToolkit.Unity
     /// </summary>
     public static class Utils
     {
+        [System.Obsolete("Use GameObjectExtensions.SetLayerRecursively(gameObject, layer) instead;")]
```

Comment:
I'm not sure what the strategy is for making methods obsolete. This is in the public Utils class so people could potentially use it in their projects. Do we just straight up remove methods or set them obsolete and delete them in 1-2 releases?